### ISSUE TITLE
Incompatible with angular.json in Angular 15

### ISSUE THREAD

**Issue by loban** [chihab/ngx-env#26]
See https://blog.ninja-squad.com/2022/11/16/angular-cli-15.0/#new-project-simplifications for the breaking change in the `polyfills` option in `angular.json`.

Because of this breaking change, after running `ng add @ngx-env/builder`, and then running `ng serve`, the following error occurs:

```
Error: Schema validation failed with the following errors:
  Data path "/polyfills" must be string.
```

This is because `polyfills` now accepts an array instead of a string.

**Comment by chihab** [comment]
Thanks for reporting it @loban. On it.

**Comment by chihab** [comment]
I just released a new version 15.0.0, could you please test it?

**Comment by loban** [comment]
> I just released a new version 15.0.0, could you please test it?

Gladly!

**Comment by loban** [comment]
> I just released a new version 15.0.0, could you please test it?

@chihab It is working perfectly! Thank you!
